architecture/additional_concepts/storage.adoc

```diff
@@ -346,10 +348,9 @@ A volume's `AccessModes` are descriptors of the volume's capabilities. They
 are not enforced constraints. The storage provider is responsible for runtime
 errors resulting from invalid use of the resource.
 
-For example, a GCE Persistent Disk has `AccessModes` *ReadWriteOnce* and
-*ReadOnlyMany*. The user must mark their claims as `read-only` if they want to
-take advantage of the volume's ability for ROX. Errors in the provider show up
-at runtime as mount errors.
+For example, a GCE Persistent Disk has *ReadWriteOnce* access mode. You must
+mark the claims as `read-only` if you want to use the volume's
+ROX capability. Errors in the provider show up at runtime as mount errors.
```

Comment:
ROX is not suitable for GCE Persistent Disk. We may need to use Ceph or iSCSI for the example.